include/pinocchio/autodiff/casadi-algo.hpp

```diff
@@ -119,6 +121,16 @@ namespace pinocchio
         }
       }
 
+      long get_fun_operation_count()
```

Comment:
```suggestion
      casadi_int get_fun_operation_count() const
```

This function modify nothing inside `AutoDiffAlgoBase` and must be const.

Reply:
Also, `get_fun_operation_count` must be in camelCase, like other method in this class.

Reply:
Thank you Joris, this is now resolved.